.hlint.yaml

```diff
@@ -0,0 +1,64 @@
+# HLint configuration file
```

Comment:
Hlint is a great tool for improving your own Haskell skill, bit should probably not be foisted on an Open Source project without first talking to the maintainers.